content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CEL in Kubernetes
+reviewers:
+- jpbetz
+- cici37
+content_type: concept
+weight: 25
+min-kubernetes-server-version: 1.25
+---
+
+<!-- overview -->
+
+The [Common Expression Language (CEL)](https://github.com/google/cel-go) is used
+in the Kubernetes API to declare validation rules, policy rules and other
+constraints or conditions.
+
+CEL expressions are evaluated directly in the API server, making CEL a
+convenient alternative to out-of-process mechanisms, such as webhooks, for many
+extensibility use cases.
+
+<!-- body -->
+
+### Language Overview
+
+The [CEL language](https://github.com/google/cel-spec/blob/master/doc/langdef.md)
```

Comment:
I'd link to https://opensource.google.com/projects/cel - how about that?

Reply:
Unfortunately that link will just redirect to the top-level `cel-spec` repo. It's probably better to preserve the link to the language definition.

Reply:
https://opensource.google.com/projects/cel is a more durable link. We can't as easily promise that https://github.com/google/cel-spec/blob/master/doc/langdef.md stays valid.